airflow-core/tests/unit/api_fastapi/core_api/test_security.py

```diff
@@ -106,7 +106,7 @@ async def test_get_user_expired_token(self, mock_get_auth_manager):
 
     @pytest.mark.db_test
     @patch("airflow.api_fastapi.core_api.security.get_auth_manager")
-    async def test_requires_access_dag_authorized(self, mock_get_auth_manager):
```

Comment:
Why did async get dropped in favour of def in all these cases? Is that related to the primary changes in this PR?